src/main/java/org/zalando/nakadi/service/timeline/TimelineService.java

```diff
@@ -165,6 +169,13 @@ public TopicRepository getDefaultTopicRepository() throws TopicRepositoryExcepti
         return topicRepositoryHolder.getTopicRepository(defaultStorage);
     }
 
+    public EventConsumer createEventConsumer(final String clientId, final List<NakadiCursor> positions)
+            throws NakadiException, InvalidCursorException {
+        final MultiTimelineEventConsumer result = new MultiTimelineEventConsumer(clientId, this, timelineSync);
+        result.reassign(positions);
```

Comment:
what is the purpose of making it a separate call? Can this be passed through constructor? 
Because I don't see any other place where 'reassign' is called.